RELEASE.CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ### July 31, 2024
 
+`2.1.1`:
```

Comment:
should it be 2.2.0 instead,  as the change doesn't look like a small patch to me